webpack/components/AnsibleRolesSwitcher/AnsibleRolesSwitcher.js

```diff
@@ -17,8 +17,12 @@ class AnsibleRolesSwitcher extends React.Component {
       inheritedRoleIds,
       resourceId,
       resourceName,
+      parentId,
+      variablesUrl,
     } = this.props.data;
 
+    this.props.initFormObjectAttrs({ resourceName, resourceId, parentId });
```

Comment:
We need to know if the form object is host or hostgroup and if it has a parent to check for inherited overrides.